src/components/notebook-utils.js

```diff
@@ -49,7 +49,18 @@ export const notebookNameValidator = existing => ({
   }
 })
 
-export const notebookNameInput = ({ inputProps, ...props }) => h(ValidatedInput, {
+// removes all paths up to and including the last slash
+export const getFileName = _.flow(_.split('/'), _.last)
+
+export const getExtension = _.flow(_.split('.'), _.last)
+
+// removes a file extension from a string
```

Comment:
For this comment and the one above, I judge the code does a better (and more maintainable) job of saying the same thing. I doubt this adds enough value to pay for the space it consumes, so I'd suggest removal.

Reply:
Yeah, was much more relevant when it was a regex